app/src/main/java/ai/haptik/android/sample/app/InboxActivity.java

```diff
@@ -100,6 +146,12 @@ public void failure(HaptikException exception) {
                 boolean isBannerVisible = view_inbox.isBannerVisible();
                 view_inbox.setBannerVisibility(!isBannerVisible);
                 break;
+            case R.id.action_haptik_offers:
+                Router.showAllOffers(this);
+                break;
+        }
+        if (intent != null) {
+            startActivity(intent);
```

Comment:
Intent is getting used at one place only

Reply:
As of now, yes! Kept it similar to how it is in our demo app. Doesn't hurt